src/App.vue

```diff
@@ -1,5 +1,14 @@
 <template>
   <div id="app" class="max-w-screen-sm lg:max-w-screen-lg mx-auto px-4 antialiased">
+    <Header />
     <router-view/>
   </div>
 </template>
+
+<script>
+import Header from '@/components/Header'
+
+export default {
+  components: { Header }
+}
```

Comment:
I will have to learn this vue stuff, looks simple enough. Name a component to be used inside of the template tags